File: Java/Booking/AddItemToBasketResult.java
package LTD_API;

public class AddItemToBasketResult
{
    private boolean success;
    private AddItemToBasketFailureReason failureReason;
    private GetBasketContentResult getBasketContentResult;
	
    public boolean getSuccess()
    {
      return success;
    }
    public void setSuccess(boolean success)
    {
      this.success = success;
    }

    public AddItemToBasketFailureReason getFailureReason()
    {
      return failureReason;
    }
    public void setFailureReason(AddItemToBasketFailureReason failureReason)
    {
      this.failureReason = failureReason;
    }

    public GetBasketContentResult getGetBasketContentResult()
    {
      return getBasketContentResult;
    }
    public void setGetBasketContentResult(GetBasketContentResult getBasketContentResult)
    {
      this.getBasketContentResult = getBasketContentResult;
    }
}

File: PHP/Booking/AddItemToBasketResult.php
<?php
namespace LTD_API;

class AddItemToBasketResult
{
    private $success;
    //instance of AddItemToBasketFailureReason
    private $failureReason;
    //instance of GetBasketContentResult
    private $getBasketContentResult;
    
    public function getSuccess()
    {
        return $this->success;
    }
    public function setSuccess($success)
    {
        $this->success = $success;
    }

    public function getFailureReason()
    {
        return $this->failureReason;
    }
    public function setFailureReason($failureReason)
    {
        $this->failureReason = $failureReason;
    }

    public function getGetBasketContentResult()
    {
        return $this->getBasketContentResult;
    }
    public function setGetBasketContentResult($getBasketContentResult)
    {
        $this->getBasketContentResult = $getBasketContentResult;
    }
}

